.github/workflows/release_droid_upload_github_release_assets.yml

```diff
@@ -1,86 +0,0 @@
-name: Release Droid - Upload GitHub Release Assets
```

Comment:
why did you remove this? we need this still for release-droid?

Reply:
I will add release related workflows etc. while addressing #3 .